src/dbnode/storage/index/types.go

```diff
@@ -60,6 +61,13 @@ type Query struct {
 	idx.Query
 }
 
+type AggregateQueryOptions struct {
+	QueryOptions
+
+	TagNameFilter      [][]byte
+	AggregateQueryType rpc.AggregateQueryType
```

Comment:
Not a fan of using rpc generated types in the exported interface.

Reply:
Yeah, agreed; was just throwing up something quick and dirty